src/frontend/src/pages/ChangeRequestDetailPage/ChangeRequestActionMenu.tsx

```diff
@@ -0,0 +1,165 @@
+import { ChangeRequest, ChangeRequestStatus, User, isLeadership, wbsPipe } from 'shared';
+import ActionsMenu from '../../components/ActionsMenu';
+import { Autocomplete, Checkbox, TextField, Box } from '@mui/material';
+import EditIcon from '@mui/icons-material/Edit';
+import DeleteIcon from '@mui/icons-material/Delete';
+import CheckBoxIcon from '@mui/icons-material/CheckBox';
+import CheckBoxOutlineBlankIcon from '@mui/icons-material/CheckBoxOutlineBlank';
+import PostAddIcon from '@mui/icons-material/PostAdd';
+import CreateNewFolderIcon from '@mui/icons-material/CreateNewFolder';
+import { useHistory } from 'react-router-dom';
+import { NERButton } from '../../components/NERButton';
+import { useRequestCRReview } from '../../hooks/change-requests.hooks';
+import { useToast } from '../../hooks/toasts.hooks';
+import { useCurrentUser } from '../../hooks/users.hooks';
+import { projectWbsPipe } from '../../utils/pipes';
+import { routes } from '../../utils/routes';
+import { userToAutocompleteOption } from '../../utils/task.utils';
+import { useState } from 'react';
+
+interface ChangeRequestActionMenuProps {
+  isUserAllowedToReview: boolean;
+  isUserAllowedToImplement: boolean;
+  isUserAllowedToDelete: boolean;
+  changeRequest: ChangeRequest;
+  handleReviewOpen: () => void;
+  handleDeleteOpen: () => void;
+  users: User[];
+}
+
+const ChangeRequestActionMenu: React.FC<ChangeRequestActionMenuProps> = ({
+  isUserAllowedToReview,
+  isUserAllowedToImplement,
+  isUserAllowedToDelete,
+  changeRequest,
+  handleReviewOpen,
+  handleDeleteOpen,
+  users
+}: ChangeRequestActionMenuProps) => {
+  const { mutateAsync: requestCRReview } = useRequestCRReview(changeRequest.crId.toString());
+  const toast = useToast();
+  const currentUser = useCurrentUser();
+  const history = useHistory();
+  const [reviewerIds, setReviewerIds] = useState<number[]>([]);
+
+  const handleRequestReviewerClick = async () => {
+    if (reviewerIds.length === 0) {
```

Comment:
I'm pretty sure you can just do an if else and then you don't need the return statement